package.json

```
@@ -7,6 +7,7 @@
     "moment": "^2.17.1",
     "sprintf-js": "^1.0.3"
   },
+  "main": "./lib/libyear-npm",
```

This file doesn't exists anymore. Did you mean: 
```suggestion
  "main": "./lib/index.js",
```